### DISCUSSION TITLE
Merci !

### DISCUSSION THREAD

**Discussion by patrowawrick** [sircharlo/jw-launcher#75] [General]
Cher frère

Excellente application, très pratique et fort utile.
Je viens de tester le programme sur Windows.
J'ai rencontré quelques bugs d'affichage mais ils sont aléatoires.
Donc difficile de les faire remonter.

Je te remercie de mettre ainsi tes compétences au service de tes fr/sr.
Avec toute mon affection fraternelle.

Patrick W
France

**Comment by sircharlo** [comment]
Merci pour les beaux mots!
